paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -1071,138 +1071,73 @@ class ActivationMKLDNNHandler
   }
 };
 
-class ReorderMKLDNNHandler : public MKLDNNHandler {
+class ReorderMKLDNNHandler {
```

Comment:
I would encourage you to extract this class to separate file. It's not a template so it would make a binary size smaller and compilation faster.